miniapps/meshing/mesh-optimizer.hpp

```diff
@@ -415,13 +415,23 @@ int material_id(int el_id, const GridFunction &g)
    double integral = 0.0;
    g.GetValues(el_id, ir, g_vals);
    ElementTransformation *Tr = fes->GetMesh()->GetElementTransformation(el_id);
-   for (int q = 0; q < ir.GetNPoints(); q++)
+   int approach = 1;
+   if (approach == 0)   // integral based
```

Comment:
If not used the `0` path can be deleted.

Reply:
Lets leave it there for future tinkering :)